modules/distribution/scripts/wso2server.sh

```diff
@@ -304,6 +304,7 @@ do
     -Djava.net.preferIPv4Stack=true \
     -Dcom.ibm.cacheLocalHost=true \
     -DworkerNode=false \
+    -Dtenant.idle.time=153722867280912 \
```

Comment:
@thiliA Just curious; why are we using 153722867280912 as tenant idle time? Have we documented the reason for using that value somewhere (couldn't find it in CEP-1345). Thanks.